setup/argocd-instance/argocd-instance.yaml

```diff
@@ -50,6 +50,27 @@ spec:
       requests:
         cpu: 250m
         memory: 256Mi
+    env:
+    - name: KUSTOMIZE_PLUGIN_HOME
+      value: /etc/kustomize/plugin
+    initContainers:
+    - args:
+      - -c
+      - cp /etc/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
+        /policy-generator/PolicyGenerator
+      command:
+      - /bin/bash
+      image: registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.6
```

Comment:
@benswinney The image version must be updated for the target RHACM version.

Reply:
We'll need to include a note within the README to capture this required change.